civictechprojects/models.py

```diff
@@ -453,7 +453,10 @@ def hydrate_to_list_json(self):
 
     @staticmethod
     def get_by_slug(slug):
-        return Event.objects.filter(event_slug=slug).first()
+        if slug is not None:
+            _slug = slug.strip()
```

Comment:
I tested my question I asked in slack just by manipulating the querystring directly, and these caused errors in console and a failure to render:
`id=5&dsfgh`
`id=Sponsor&Sponsor` 
and similar uses of ? and &.

this is different from a numerical ID which doesn't exist or a string without special characters, like `id=65` or `id=fakeslug` - both of which correctly render the page and output the message "Could not load Event" as expected.

As slug creation is (I believe?) admin side only right now we can just make sure not to create bad slugs in the short term, but eventually an automatic system (lowercase only replacing all special characters with dashes is the most common pattern I've seen) would help prevent errors here.